### DISCUSSION TITLE
Localhost:5173 本地开发的应用（部署之前）可以直接打包么？

### DISCUSSION THREAD

**Discussion by SaraiQX** [tw93/Pake#370] [Q&A]
如题。Web应用部署到阿里云或其他hosting有点麻烦（域名解析啥的，小白泪奔），可以直接打包么？之前用Tauri尝试是可以的呢。
感谢大神！

**Comment by SaraiQX** [comment]
自问自答😂 ：去试了一下，在npm run dev后，应用测试正常，直接用了官网guideline 命令行输入”pake http://localhost:5173/ --name myapp --transparent“，几分钟后成功打包为dmg文件（本人Mac）。本地安装OK，注册登录OK，只是网页中通过iframe嵌入的网页打不开😭（算了还是得自己开发呀，看起来(伪)”混合技术栈“行不通的。

**Reply by I7T5** [reply]
如果网页里没有iframe可以吗😂 小白+1，想写个web app分享给朋友，但出于隐私考虑希望数据全本地。

**Reply by SaraiQX** [reply]
当然可以啊，iframe是我自己加的，为了用其他组件。注意就是分享给别人的时候，你的电脑也要开机、APP（包括数据库）正常运行、别人才能打开

**Reply by I7T5** [reply]
Hmm, 好吧, 那看来我也没法省事了
